lib/registration.pm

```diff
@@ -182,6 +182,20 @@ sub assert_registration_screen_present {
     }
 }
 
+sub verify_preselected_modules {
+    my @modules = ('basesystem', 'server', split(/,/, get_var('ADDONS', '')));
+    my @needles = map { 'addon-' . $_ } @modules;
+    for (1 .. 10) {
```

Comment:
God solution! The only thing, 10 is actually not enough, for some scenarios we have 50 (see https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/registration.pm#L196 ) So I would suggest even extracting this magic number as constant and reusing, if you don't mind.

Reply:
not sure if both numbers are referring to the same thing, one could be the number of modules(?) and for this case 10 is the number of scroll down increments (which is not the same than to move in the list 50 times) which is faster as far I could test manually. But anyway it doesn't hurt to refactor it for a few more screenshots to play safer.

Reply:
You are right, now I somehow remembered that it was magic number 40 and not 50 (maybe because I've increased it), piece of code I was referring to is https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/registration.pm#L342

Reply:
ok, I can refactor that part with the 40, I get it now. thx

Reply:
So we don't need it in this loop? I got confused now, sorry =)